src/main/java/nextmainfocus/account/SecurityConfiguration.java

```diff
@@ -44,10 +44,10 @@ public InMemoryUserDetailsManager userDetailsManager() {
 	}
 
 	@Bean
-	public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
-		http.authorizeHttpRequests().requestMatchers("/healthcheck", "/api/v1/login/").permitAll().anyRequest()
+	SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
+		http.authorizeHttpRequests().requestMatchers("/",
+				"/api/v1/login/").permitAll().anyRequest()
```

Comment:
should the url be hard-coded at this level?

Reply:
Yes, afaik.